include/ringmesh/geogram_extension/geogram_extension.h

```diff
@@ -140,31 +141,31 @@ namespace RINGMesh
      * @todo Probably extremely prone to bugs. Is it worth the risk?
      */
     template < typename T >
-    class AttributeVector : public std::vector< GEO::Attribute< T >* >
```

Comment:
Is this class still necessary since we can have Atributes of vectors? Is yes, we have to move this class into another file.

Reply:
I do not know. Maybe we could remove it.